src/scripts/shell/platform-choice/platform-choice.component.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
```

Comment:
Lodash is being improved.  Last release of underscore is in 2015.  I'm bringing in just the functions I need.  This way _ can still be used by underscore, and the lodash functions I need are brought in individually.

This is one of those features of imports/exports & tree-shaking that lets you use just the code you need.  Compilers can infer it or you can be explicit.